release-notes/1.20.0.md

```diff
@@ -0,0 +1,63 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.20.0.
+
+Please see [component versions](https://anbox-cloud.io/docs/ref/component-versions) for a list of updated components.
+
+## New features & improvements
+
+### Core stack improvements
+
+* Anbox Cloud has introduced support for creating virtual machine instances. This is especially helpful for use cases that require a stronger isolation. You can use both containers and virtual machines for your applications.
+
+  For easy understanding, we have introduced a change in terminology as well. The *Anbox Cloud containers* or *LXD containers* are now called *Instances*. The term *Instance* in the [Anbox Cloud documentation](https://anbox-cloud.io/docs) denotes a container or a virtual machine that hosts an application or an image. <!--AC-1459 and AC-2010 and AC-1916-->
```

Comment:
I feel like it's worth mentioning the creation of GPU based VM instance is unsupported in 1.20.0 release.

Reply:
Done, thanks!